src/XCCDF_POLICY/xccdf_policy_model.c

```diff
@@ -46,6 +47,34 @@ struct xccdf_policy *xccdf_policy_model_get_existing_policy_by_id(struct xccdf_p
 	return NULL;
 }
 
+static void _add_selectors_for_all_xccdf_items(struct xccdf_profile *profile, struct xccdf_item *item)
+{
+	struct xccdf_item_iterator *children = NULL;
+	if (xccdf_item_get_type(item) == XCCDF_BENCHMARK) {
+		children = xccdf_benchmark_get_content(XBENCHMARK(item));
+	}
+	else if (xccdf_item_get_type(item) == XCCDF_GROUP) {
```

Comment:
The coding style https://www.kernel.org/doc/html/v4.10/process/coding-style.html prefers
```
if (x == y) {
        ..
} else if (x > y) {
        ...
} else {
        ....
}
```

Reply:
Fixed